lib/client-assets.php

```diff
@@ -95,6 +95,11 @@ function gutenberg_override_script( $scripts, $handle, $src, $deps = array(), $v
 	if ( 'wp-i18n' !== $handle && 'wp-polyfill' !== $handle ) {
 		$scripts->set_translations( $handle, 'default' );
 	}
+	if ( 'wp-i18n' === $handle ) {
+		$ltr    = _x( 'ltr', 'text direction', 'default' );
```

Comment:
Are translations considered safe? This is outputting unescaped text from translations into JavaScript quite directly. 

Is there some escaping or cleaning we can do? Maybe something like this is good, since our expected strings are well known:

```suggestion
		$ltr    = 'rtl' === _x( 'ltr', 'text direction', 'default' ) ? 'rtl' : 'ltr';
```

Alternatively, this seems idiomatic via [`is_rtl()`](https://developer.wordpress.org/reference/functions/is_rtl/).

```suggestion
		$ltr    = is_rtl() ? 'rtl' : 'ltr';
```

Reply:
I'd prefer to keep the current behavior personally because it matches the expectation that  the translators in glotpress can decide what to do here. And about the safety, I'm not sure. I noticed that core uses `load_script_textdomain` which loads the translations from a json file and just paste the JSON as is to this function.